docs/source/ultimate.rst

```diff
@@ -59,6 +59,70 @@ Nous garderons la partie privée de vos données (nom, coordonnées, etc.) dans
+
+A nuestro entender, los datos que publicamos en abierto en general no contendrán información que puede identificarle directamente. La Base de Datos generada no contendrá su nombre, solo un código, de forma que el público no sabrá ni su nombre ni que los datos son suyos. Además, los datos no van a incluir información que pensemos podrían ayudar a personas cercanas a usted a adivinar qué los datos son suyos, como rasgos faciales o el día que participo. Si escribimos un reportaje o artículo sobre el estudio o compartimos los datos con otros, lo haremos de manera que usted no podrá ser identificado directamente. Sin embargo, usando datos adicionales relacionados con su nombre (por ejemplo, un escaneo cerebral obtenido de su historia clínica), uno posiblemente podría asociar su imagen u otra información en nuestra base de datos con usted. Además, si se produce una violación de seguridad (ataque cibernético) que pudiera hacer qué alguien le asociarse a usted con sus datos. Este riesgo es muy bajo porque sus datos están guardados en una base de datos segura, y la información sobre usted está guardado por separado, relacionados solo con un código.
+
+Mantendremos la parte privada de sus datos (nombre, información para contactarle, etc.) en un lugar seguro durante al menos <x> años. De esta manera, si uno de los investigadores que utilice los datos de este estudio encuentra algo en sus escaneos cerebrales que pudieran tener un valor diagnóstico, podremos contactar con usted. Después de este periodo de tiempo, destruiremos su información para proteger su privacidad.
+
+**Dejarnos usar y compartir sus datos es voluntario. Sin embargo, tiene que estar dispuesto a compartir sus datos de esta forma para poder participar en este estudio.**
+
+Si no está dispuesto, no podrá participar en el estudio. Al firmar abajo, está aceptando proporcionar sus datos a futuras investigaciones. De esta forma acepta que los datos podrán ser compartidos con otros investigadores de otras instituciones por todo el mundo. Los detalles, resultados, e implicaciones de estos estudios son desconocidos.
+
+Italian
+~~~~~~
+(translation courtesy of Valentina Borghesani)
+
+I dati ed i campioni raccolti in questo studio potrebbero essere utilizzati per altri futuri progetti di ricerca oltre allo studio a cui stai attualmente partecipando. Tali ricerche potranno riguardare qualsiasi argomento, non necessariamente collegato a quello dello studio cui stai partecipando. I dati raccolti, incluse le neuroimmagini, saranno condivisi attraverso una piattaforma online ed una banca dati aperta al grande pubblico.
```

Comment:
How about using the "lei" throughout the document? I'm not used to consent forms in Italian any more, so disregard this in case ;)

Other things:
```
neuroimmagini -> immagini di risonanza magnetica
aperta al grande pubblico -> accessibile pubblicamente
```

Reply:
Thanks @mvdoc ! 

I too was wondering whether to go formal with "lei" or not. Let's go for it! (some sentences will be re-written accordingly)

I'll implement your suggestions now (thanks! very rusty italian here too), the only one I don't fully agree with is "neuroimmagini -> immagini di risonanza magnetica", what about other modalities? Shouldn't we leave the field open for M/EEG data etc... ?